h2o-persist-s3/src/main/java/water/persist/PersistS3.java

```diff
@@ -31,10 +32,10 @@
 
 /** Persistence backend for S3 */
 public final class PersistS3 extends Persist {
-  private static final String HELP = "You can specify a credentials properties file with the -aws_credentials command line switch.";
 
   private static final String KEY_PREFIX = "s3://";
   private static final int KEY_PREFIX_LEN = KEY_PREFIX.length();
+  private static final Logger LOGGER = Logger.getLogger("water.default");
```

Comment:
@honzasterba Do we want to use this convention everywhere ? I'd put proper class in there.